### DISCUSSION TITLE
Welcome to Discord-AI-Selfbot Discussions!

### DISCUSSION THREAD

**Discussion by Najmul190** [Najmul190/Discord-AI-Selfbot#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by pnydiffuser** [comment]
I sense a new mega update soon, the day it is here is gonna be lit.

**Reply by pnydiffuser** [reply]
My question is if the analyse command will carry over to  the next update? there was a few people who wanted to try it as gpt ai has improved over all that time.

**Reply by Najmul190** [reply]
should work when i release the latest update
